helpdesk_mgmt_crm/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    lead_id = fields.Many2one(comodel_name="crm.lead", string="Opportunity")
```

Comment:
This should be a one2many.

Reply:
I wouldn't restrict it. Imagine a customer asks for 3 products at the same time and you want to redirect each product to one salesman...

Reply:
For supporting this case, let's copy the messages instead of transferring them.